release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1900,6 +1900,8 @@ For more information about early access, xref:../updating/understanding_updates/
 
 * Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
 
+* In hosted control planes for {product-title}, on the bare metal platform, if a worker node fails, another node is not added to the hosted cluster, even when other worker nodes are available. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```

Comment:
```suggestion
* In hosted control planes for {product-title}, on the bare metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```

Small suggestion and added a note for a workaround. Feel free to adjust the language. Thanks!